charts/gsp-cluster/templates/04-main-team-pipelines/cd-smoke-test-pipeline.yaml

```diff
@@ -71,6 +71,7 @@ spec:
         trigger: true
       - get: src
       - put: image
+        get_params: {skip_download: true}
```

Comment:
Is this not a useful verification test that the image was uploaded successfully and can be pulled?

Reply:
🤷‍♂ if that's the intention, maybe it should be documented, otherwise someone like me will probably come along in future and remove it